docs/content/en/docs/clustermgmt/security/manually-renew-certs.md

```diff
@@ -168,3 +168,74 @@ ${IMAGE_ID} tmp-cert-renew \
 5. Repeat the above steps for all control plane nodes.
 
 You can similarly use the above steps to rotate a single certificate instead of all certificates.
+
+#### Update kubeconfig on Admin machine
+
+Post completion of manual certificate rotation or if your Cluster was created more than a year ago, your kubeconfig file on Admin machine will have outdated certificates and would result in following error,
+
+```
+Error: Couldn't get current Server API group list: the server has asked for the client to provide credentials error: you must be logged in to the server.
+This error typically occurs when the cluster certificates have been renewed or extended during the upgrade process. To resolve this issue, you need to update your local kubeconfig file with the new cluster credentials.
+```
+
+Follow below steps to update kubeconfig on Admin machine.
+
+1. SSH to one of the Control Plane nodes and run the following command to validate connection with API Server, export kubeconfig from `${CLUSTER_NAME}-kubeconfig` secret object (`eksa-system` namespace) using kubectl and copy kubeconfig file to `/tmp` directory.
```

Comment:
This step is only necessary if the local kubeconfig has already expired. If not, they should be able to retrieve the secret with the existing cert data.

Can we move the step to retrieve the admin.conf to a separate subheading "What do I do if my local kubeconfig has expired?"